JavaScript/Topics/18.Rollovers/Rollovers.md

```diff
@@ -0,0 +1,173 @@
+<br>
+<a onmouseover=”document.book.src=’vb6book.jpg'”><b>Visual Basic 6 Made Easy</b></a>
+<br>
+</td>
+</tr>
+</tbody>
+</table>
+</body>
+</html>
+
+```
+## Output 
+
+<br>
+
+![image](op-3.PNG)
+<br><br>
+<br>
+
+![image](op-4.png)
+<br><br>
+
+<br>
+
+![image](op-5.png)
+<br><br>
+
+
+# **Text Rollover**
+ You can create as many rollovers as you want on your web page; however,each one should be meaningful to the visitor. There is nothing more distracting to a visitor than to encounter rollovers on practically every object on a web page. Carefully placed rollovers can enhance a visitor's experience when browsing the web page. A clever rollover technique used by some developers is to enable a visitor to see additional information about an item described in text by placing the mouse cursor on the text. This eliminates the time-consuming task of using a hyperlink to display another web page that contains this additional information and reduces the information clutter found on some web pages.You create a rollover for text by using the **onmouseover** attribute.
+
+
+## **Example :**
+
+### In example below, we create a rollover effect that can change the text when move goes over the text.
+
+```JavaScript
+<HTML>
+<head></head>
+<Body>
+<textarea rows=”2″ cols=”50″ name=”rollovertext” onmouseover=”this.value=’What is rollover?'”
+onmouseout=”this.value=’Rollover means a webpage changes when the user moves his or her mouse over an object on the page'”></textarea>
+</body>
+</html>
+```
+
+## Output
+
+<br>
+
+![image](op-6.png)
+<br><br>
+
+<br>
+
+![image](op-7.PNG)
+<br><br>
+
+# **More Efficient Rollovers**
+An efficient way of handling rollovers is to load images into an array when your 
+web page loads. The browser loads each image once the fi rst time the image is referenced in the web page. Typically, the default setting for the browser is to check 
+the browser cache for subsequent references for the image rather than download the 
+image again from the web server. However, a visitor to your web page might have 
+changed the default setting, causing the browser to reload the image each time the 
+image is referenced. This might cause a noticeable delay.
+Any delay in transmission is likely to be noticed by the visitor. While most visitors accept short delays when they're selecting a different web page, they tend to be 
+unforgiving if the rollover takes longer than a second or two to display the new image. You can reduce this delay by creating a JavaScript that loads all the images into 
+memory once at the beginning of the JavaScript, where they can be quickly called 
+upon as the onmouseover event occurs.
+
+Downloading images when the web page is f rst loaded is a simple three-step 
+process:
+ 1. Declare an image object.
+ 2. Assign the image fi le to the image object.
+ 3. Assign the image object to the src attribute of the HTML tag that is going 
+to react to the rollover event.
+
+
+## **Summary**
```

Comment:
@utkarshakakade  Please short the summary as much as possible:-)

Reply:
Sure!